stable/coredns/README.md

```diff
@@ -80,3 +80,14 @@ $ helm install --name coredns -f values.yaml stable/coredns
 ```
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
+
+
+Caveats
+-------
+
+CoreDNS service, by default is deployed to listen on both "TCP" and "UDP".
```

Comment:
It might be good to have this notice or a similar one in NOTES.txt as well if the serviceType=LoadBalancer and the serviceProtocol= UDPNTCP